webaverse/app
# New grab animations

Related issues:
https://github.com/webaverse/app/issues/1981
https://github.com/webaverse/app/issues/1873
https://github.com/webaverse/app/issues/1825

In this PR we have new animations for item grab action.
Animations are: 
grab forward
grab down
grab up
grab left
grab right

Also "group objects cannot be found when worn" is already solved by Avaer but we just need one simple null check and lantern/fairy can be picked up / droped without any problem.

Steps for testing:
Prepare scene with pickable objects at different heights.
Check object grab / pick up in directly in front of player, on players left side, on players right side.
Also place pickable object above players head, and on the floor. 
Appropriate animation should be shown.

Plase take a look at video below for reference
